### ISSUE TITLE
Too much logging on CircleCI

### ISSUE THREAD

**Issue by brianmhunt** [lerna/lerna#812]
When running lerna on CircleCI, the logs are polluted with tons of lines like this:

```
[..................] | exec: verb rootPath /home/ubuntu/tko
[..................] | exec: verb rootPath /home/ubuntu/tko
```

The command being run is `npm test`, where `test` is defined as `lerna exec --concurrency=1 --loglevel=warn -- karma start ../../karma.conf.js --once `

## Expected Behavior

To be able to turn this logging off off.  

## Current Behavior

Too much logging.

## Possible Solution

Most sensible options would probably be

- respecting `--loglevel` of warn
- automatically detecting the absence of a tty (`process.stdout.isTTY`)
- an explicit `--quiet` argument

## Steps to Reproduce (for bugs)

Submit something to CircleCI

## Context

If so inclined, reproduce with github.com/knockout/tko

## Your Environment

Ubuntu 14.04 on CircleCI with Node 7. with lerna 2.0.0-rc.4

**Comment by evocateur** [comment]
This was fixed in #798, which has not been released yet. There are a couple of PRs I was hoping to land before cutting a proper `v2.0.0`.

**Comment by brianmhunt** [comment]
Cool, thanks for the heads up.  Sorry I didn't see the PR when searching.
